src/panels/lovelace/hui-editor.ts

```diff
@@ -35,7 +41,12 @@ class LovelaceFullConfigEditor extends hassLocalizeLitMixin(LitElement) {
               @click="${this.closeEditor}"
             ></paper-icon-button>
             <div main-title>Edit Config</div>
-            <paper-button @click="${this._handleSave}">Save</paper-button>
+            <paper-button @click="${this._handleSave}"
+              >Save<ha-icon class="${classMap({
+                saved: this._saving! === false,
+                "save-button": true,
```

Comment:
Actually, we should also be able to do `class="save-button ${classMap({ saved: … })}"`

Reply:
I feel like I have read somewhere that its better to use only class map. But thinking about it that doesn't *really* makes sense. So will change